crates/lang/codegen/src/generator/cross_calling.rs

```diff
@@ -685,7 +685,7 @@ impl CrossCalling<'_> {
             fn #ident(
                 #( #input_bindings : #input_types ),*
             ) -> Self::#output_ident {
-                ::ink_env::call::build_create::<Environment, Salt, Self>()
+                ::ink_env::call::build_create::<Environment, Self>()
```

Comment:
It has been added during Salt feature, but it is not used here and causes a compilation error.

Reply:
It shouldn't case problems with respect to this PR. Just don't remove the `Salt` parameter.

Reply:
Okey. Do you plan to remove this Salt later? Because it doesn't allow to import contract with `ink-as-dependency` if it implements constructor during the implementation of trait(it cause compilation error).

Reply:
I don't really understand the problem as stated so far. :S

Reply:
Okey=) Let's try another description=D
[build_create](https://github.com/paritytech/ink/blob/master/crates/env/src/call/create_builder.rs#L198) method supports only 2 generic parameters. But code here is passing 3 parametrs=) It causes a compilation error. Above I described a case when ink! will generate this code which can't be compiled